src/com/emailage/javawrapper/model/ExtraInputParameter.java

```diff
@@ -23,6 +23,16 @@ public void setlastname(String value) {
 		lastname = value;
 	}
 
+	private String billaddress = new String();
+
+	public String getbilladdress() {
+		return billaddress;
+	}
+
+	public void setbilladdress(String value) {
+		billcity = billaddress;
```

Comment:
Tony, can you explain this change? Are you setting the bill city or the billing address.

Reply:
Good catch!  It is now fixed..   billaddress = value;